converter/bodies.py

```diff
@@ -0,0 +1,91 @@
+from dataclasses import dataclass
+from abc import ABC
+
+
+@dataclass(frozen=True)
+class Body(ABC):
+    """Abstract body, bodies are used to define geometries."""
+
+    uuid: str
+    x_offset: float
+    y_offset: float
+    z_offset: float
+    x_rotation: float
+    y_rotation: float
+    z_rotation: float
+
+
+@dataclass(frozen=True)
+class SphereBody(Body):
+    """
+    A sphere body. It's defined by its radius, offset from the point (0, 0, 0) xyz
+    and its rotation along each axis.
+    """
+
+    radius: float
```

Comment:
Sphere inherits rotation from its base class which doesn't make sense.

Reply:
As we discussed: for now sphere will have rotation, but it won't be used.

Reply:
ok